modules/ducktests/.gitignore

```diff
@@ -0,0 +1,2 @@
+tests/venv/
```

Comment:
I believe that there is no need to  add .gitignore here at all.
And I strongly advice you to use virtualenvwrapper in order do maintain different venv's.
It is recommended way in python community to use venv's in developer's environment.
If you used it it would not be necessary add something to .gitignore.